spec/system/admin/invoice_print_spec.rb

```diff
@@ -449,6 +449,221 @@
     it_behaves_like "contains right Payment Description at Checkout information"
     it_behaves_like "Check display on each invoice: legacy and alternative", false
     it_behaves_like "Check display on each invoice: legacy and alternative", true
+    describe "order with taxes" do
+      let(:user1) { create(:user, enterprises: [distributor]) }
+      let!(:zone) { create(:zone_with_member) }
+      let(:address) { create(:address) }
+
+      context "included" do
+        let(:shipping_tax_rate_included) {
+          create(:tax_rate, amount: 0.1, included_in_price: true, zone:)
+        }
+        let(:enterprise_fee_rate_included) {
+          create(:tax_rate, amount: 0.15, included_in_price: true, zone:)
+        }
+        let(:shipping_tax_category) {
+          create(:tax_category, tax_rates: [shipping_tax_rate_included])
+        }
+        let(:fee_tax_category) {
+          create(:tax_category, tax_rates: [enterprise_fee_rate_included])
+        }
+        let!(:shipping_method_name) { "SM1" }
+        let!(:shipping_method) {
+          create(:shipping_method_with, :expensive_name, name: shipping_method_name, distributors: [distributor],
+                                                         tax_category: shipping_tax_category)
+        }
+        let!(:enterprise_fee) {
+          create(:enterprise_fee, enterprise: user1.enterprises.first,
+                                  tax_category: fee_tax_category,
+                                  calculator: Calculator::FlatRate.new(preferred_amount: 120.0))
+        }
+        let!(:order_cycle) {
+          create(:simple_order_cycle,
+                 coordinator: distributor,
+                 coordinator_fees: [enterprise_fee],
+                 distributors: [distributor],
+                 variants: [product1.variants.first, product2.variants.first])
+        }
+
+        let!(:order1) {
+          create(:order, order_cycle:, distributor: user1.enterprises.first,
+                         ship_address: address, bill_address: address)
+        }
+        let!(:product1) {
+          create(:taxed_product, zone:, price: 12.54, tax_rate_amount: 0,
+                                 included_in_price: true)
+        }
+        let!(:product2) {
+          create(:taxed_product, zone:, price: 500.15, tax_rate_amount: 0.2,
+                                 included_in_price: true)
+        }
+
+        let!(:line_item1) {
+          create(:line_item, variant: product1.variants.first, price: 12.54, quantity: 1,
+                             order: order1)
+        }
+        let!(:line_item2) {
+          create(:line_item, variant: product2.variants.first, price: 500.15, quantity: 3,
+                             order: order1)
+        }
+
+        let(:url_params) { {} }
```

Comment:
The setup for this test looks the same as the old test, I wonder if it can be shared?  It could make the code much easier to manage.

Or perhaps it doesn't matter if we expect to delete the old test soon anyway?